addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -162,9 +167,9 @@ var SlideUploadDialog = Dialog.extend({
                     'mime_type': this.file.type === 'image/svg+xml' ? 'image/png' : this.file.type,
                     'datas': this.file.type === 'image/svg+xml' ? this._svgToPng() : this.file.data
                 });
-            } else {
+            } else{
                 _.extend(values, {
-                    'image': this.file.type === 'image/svg+xml' ? this._svgToPng() : this.file.data,
+                    'image_1920': this.file.type === 'image/svg+xml' ? this._svgToPng() : this.file.data,
```

Comment:
Yeah, it's because some guys thought it was a good idea to rename image into image_1920, image_small into image_640 (or something) etc.. :roll_eyes: 

Well, was it already the case in 13? if the fix should be done in 13 too, first do a fix in 13, either wait for the fix to be merged and forward ported in master, either cherry-pick the v13 fix commit in this branch.